### DISCUSSION TITLE
You use my project as a dependency.

### DISCUSSION THREAD

**Discussion by its-danny** [WordPress/gutenberg#64968] [General]
Hey there. You use a library I made (https://github.com/its-danny/use-lilius) as a dependency for your date picker. I think that's pretty cool.

That said, it's basically unmaintained. I haven't had a need for it for some years and am generally trying to move my career away from React things, so I don't have a lot of interest in maintaining it myself.

It seems to be functioning just fine, but I'm in the process of cleaning up my account and was hoping to find someone to give ownership of it to and I figured I'd ask y'all.

Regardless of if you want it or not, thanks for using it! It makes me feel all tingly inside.

**Comment by ciampo** [comment]
Hey @its-danny, thank you for reaching out, we appreciate the heads-up!

> I haven't had a need for it for some years

Out of curiosity, have you found alternatives that better suited your use cases?

> It seems to be functioning just fine, but I'm in the process of cleaning up my account and was hoping to find someone to give ownership of it to and I figured I'd ask y'all.

Thank you for the offer! I'm not sure if we have an appetite for owning and maintaining the project, but we could make a local copy and keep it internal to `@wordpress/components` so that we don't link the the `npm` dependency anymore.

cc @WordPress/gutenberg-components @noisysocks @t-hamano

**Reply by DaniGuardiola** [reply]
react-day-picker is a good one

**Reply by jsnajdr** [reply]
The package is one file with one hook that maintains calendar state. Renders no UI. A local copy seems to be the best option.

**Reply by tyxla** [reply]
Agreed on a local copy 👍

**Reply by ciampo** [reply]
https://github.com/WordPress/gutenberg/pull/65097
